src/ralph/security/api.py

```diff
@@ -80,6 +83,19 @@ class SecurityScanViewSet(RalphAPIViewSet):
     serializer_class = SecurityScanSerializer
     save_serializer_class = SaveSecurityScanSerializer
 
+    def _filter_by_ip(self, queryset):
```

Comment:
use `FilterSet`, similar to https://github.com/allegro/ralph/blob/ng/src/ralph/assets/api/filters.py#L4
Using this, this field will appear in OPTIONS result